<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="start">
     <widget class="QLabel" name="title_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>274</width>
        <height>75</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>40</pointsize>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>MealMate</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>JetBrains Mono</family>
        <pointsize>9</pointsize>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>meal planning and grocery tracking app</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startExitButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>200</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>200</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="main">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>199</green>
            <blue>158</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>171</red>
            <green>199</green>
            <blue>158</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled/>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="gridVisible">
       <bool>false</bool>
      </property>
      <property name="horizontalHeaderFormat">
       <enum>QCalendarWidget::HorizontalHeaderFormat::SingleLetterDayNames</enum>
      </property>
      <property name="verticalHeaderFormat">
       <enum>QCalendarWidget::VerticalHeaderFormat::NoVerticalHeader</enum>
      </property>
      <property name="navigationBarVisible">
       <bool>true</bool>
      </property>
      <property name="dateEditEnabled">
       <bool>true</bool>
      </property>
      <property name="dateEditAcceptDelay">
       <number>1500</number>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="editButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>520</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>edit</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>371</width>
        <height>201</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="meal1_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>331</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="meal2_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>331</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="meal3_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>331</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>-10</y>
        <width>167</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>MealMate</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>301</width>
        <height>471</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>299</width>
         <height>469</height>
        </rect>
       </property>
       <widget class="QLabel" name="ingredientsList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>281</width>
          <height>451</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>71</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Meals</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>136</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Ingredients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generateButton">
      <property name="geometry">
       <rect>
        <x>599</x>
        <y>520</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>generate list</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="invButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>0</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="text">
       <string>inventory</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="edit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>691</width>
        <height>551</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Meal 1</string>
       </attribute>
       <widget class="QTextEdit" name="inglist1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>lettuce 1, tomato 2, salad_dressing 1, ...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>97</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>meal title</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>118</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ingredients</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="mealtitle1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>641</width>
          <height>51</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Meal 2</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>118</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ingredients</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>97</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>meal title</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="mealtitle2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>641</width>
          <height>51</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="inglist2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>lettuce 1, tomato 2, salad_dressing 1, ...</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Meal 3</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>97</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>meal title</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="mealtitle3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>641</width>
          <height>51</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Poppins'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="inglist3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>lettuce 1, tomato 2, salad_dressing 1, ...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>130</y>
          <width>118</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ingredients</string>
        </property>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>470</y>
          <width>167</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <bold>false</bold>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="inventory">
     <widget class="QListWidget" name="invList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>701</width>
        <height>431</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="inventoryTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>164</width>
        <height>47</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="toolTip">
       <string>select an ingredient to remove</string>
      </property>
      <property name="text">
       <string>remove</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invBackButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>510</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
